src/awesome_map/AwesomeMap.js

```diff
@@ -927,7 +928,7 @@ define(function(require) {
                 return true;
             }
 
-            if (!this._options.touchScrollingEnabled && (
+            if (!this._options.mousePanningEnabled && EventSource.isMouse(event) && (
```

Comment:
Notable change here:  We now only consider the event cancelled if mouse panning is off and it is a mouse event.

Previously, if mouse panning (touchScrollingEnabled) was disabled, we could not pan with touch events, either.